src/authentication/select-method/index.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+import { Button } from '@zero-tech/zui/components';
+
+import './styles.scss';
+import { bem } from '../../lib/bem';
+const c = bem('select-method');
+const cn = (m?, s?) => ({ className: c(m, s) });
+
+export interface Properties {
+  onEmailSelected: () => void;
+  onWalletSelected: () => void;
+}
+
+export class SelectMethod extends React.PureComponent<Properties> {
+  render() {
+    return (
+      <div {...cn()}>
+        <h3 {...cn('heading')}>How do you want to create an account?</h3>
```

Comment:
Experimenting here with this classname shortcut. It's quite a bit shorter but, obviously, is non-standard so it takes a moment to understand what's going on if you haven't seen it before. Trying to find good ways to reduce the overhead of scanning a component's structure versus being able to "grep" the code for class names, etc.

```
<h3 {...cn('heading')}>
<h3 className={c('heading')}>
```